src/lib/bind/accessors.html

```diff
@@ -163,7 +163,27 @@
           return this.__data__[property];
         }
       };
+
+
       var setter = function(value) {
+        // TODO: Am I allowed to call to micro from the core lib like this?
```

Comment:
Am I allowed to call to to micro from the core lib like this? I noticed that properties is also in micro so is it safe to assume that micro is always imported? Furthermore, is this a feature that even makes sense being in micro? It doesn't seem like 'bare minimum custom element sugaring' it seems more like it should be a part of polymer standard but then i'm calling to standard all the way from the core lib.